lib/dependency.js

```diff
@@ -13,7 +13,7 @@ Dependency.prototype.cloneShallow = function () {
 };
 
 Dependency.extractFromDotVersionNotation = function (expression) {
-  var versionRef = /(?=\S+)(?=\.{1})((\.\d+)+)/.exec(expression)[0];
```

Comment:
here's all that has changed. The rest it renaming some dependencies to follow pre release semantics and 2.0.0 semantics